### PR TITLE
Ignore latest rustsec issue

### PR DESCRIPTION
**Pull request by DaughterOfMars**
# Description of change

Ignore the audit

### CODE REVIEW

**Review by thibault-martinez — DISMISSED** [review]

File: .cargo/config.toml [hunk 1]
```diff
@@ -21,4 +21,6 @@ ci-license = "license-template --template .license_template"
 #
 # - RUSTSEC-2021-0065: https://rustsec.org/advisories/RUSTSEC-2021-0065
 #   - anymap is unmaintained 🤷‍♀️
-ci-audit = "audit --file Cargo.lock --deny warnings --ignore RUSTSEC-2021-0065"
+# - RUSTSEC-2023-0052: https://rustsec.org/advisories/RUSTSEC-2023-0052
```
Comment (abdulmth) [inline .cargo/config.toml:24]: can we maybe add a "todo"?

**Review by abdulmth — DISMISSED** [review]

File: .cargo/config.toml (hunk 1 above)
Comment (DaughterOfMars) [reply .cargo/config.toml:24]: sure, added

**Review by abdulmth — DISMISSED** [review]

**Review by kwek20 — DISMISSED** [review]

File: .cargo/config.toml [hunk 2]
```diff
@@ -21,4 +21,7 @@ ci-license = "license-template --template .license_template"
...
+# - RUSTSEC-2023-0053: https://rustsec.org/advisories/RUSTSEC-2023-0053
+#   - TODO: waiting for fix in dependency
+ci-audit = "audit --file Cargo.lock --deny warnings --ignore RUSTSEC-2021-0065 --ignore RUSTSEC-2023-0052  --ignore RUSTSEC-2023-0053"
```
Comment (thibault-martinez) [inline .cargo/config.toml]: I don't think we should add 53 :(
It was already gone if we updated the deps

Comment (DaughterOfMars) [reply .cargo/config.toml]: ugh fiiine

**Review by thibault-martinez — DISMISSED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Pull request merged**